crates/napi/types/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.ts
```

Comment:
GH Copilot told me that 

> This means that all TypeScript files will be ignored except for node-types.ts.

Is this a desired change? I think the goal here is to include ts file except node-types?

Reply:
My goal was to not to check the language-specific node types TS files in the repo as it's "auto-generated/downloaded". I don't have a strong opinion whether we should do that or not.

Reply:
<img width="255" alt="Screenshot 2024-12-14 at 11 44 34 AM" src="https://github.com/user-attachments/assets/984f00c3-f3ae-4c75-b3a7-e2d57ba21d7b" />

Reply:
and `node-types` without language name prefix is the type utils to extract possible field names, etc.

Reply:
Oh sorry I misread the .gitignore directory. It's under types and I thought it was under `napi`